packages/server/graphql/types/Task.ts

```diff
@@ -103,14 +112,96 @@ const Task = new GraphQLObjectType<any, GQLContext>({
                   }
                 }`
           const githubRequest = (info.schema as any).githubRequest as GitHubRequest
-          const {data, errors} = await githubRequest({
-            query,
-            endpointContext,
-            batchRef: context,
-            info
-          })
-          if (errors) {
-            console.log(errors)
+
+          const [{data, errors}, {data: labelsData, errors: labelErrors}] = await Promise.all([
+            githubRequest({
+              query,
+              endpointContext: {
+                accessToken: githubAuth.accessToken
+              },
+              batchRef: context,
+              info
+            }),
+            estimates.length > 0
+              ? githubRequest<GetIssueLabelsQuery, GetIssueLabelsQueryVariables>({
+                  query: getIssueLabels,
+                  variables: {
+                    first: 100,
+                    repoName,
+                    repoOwner,
+                    issueNumber
+                  },
+                  endpointContext: {
+                    accessToken: githubAuth.accessToken
+                  },
+                  batchRef: context,
+                  info
+                })
+              : {data: null, errors: null}
+          ])
+
+          if (errors || labelErrors) {
+            if (errors) {
+              console.error(errors)
+              sendToSentry(new Error(errors[0].message), {
+                userId: accessUserId
+              })
+            }
+            if (labelErrors) {
+              console.error(labelErrors)
+              sendToSentry(new Error(labelErrors[0].message), {userId: accessUserId})
+            }
+          } else if (estimates.length) {
+            const labels = labelsData.repository.issue.labels.nodes
```

Comment:
@mattkrick thanks, added new fixes

I only removed this filter:
``` 
.filter((row) => row.ne(taskId))
```

As otherwise, we can't pick the labels that were used in past but only on the current task. 
And it is not possible to pick the current label anyway as we check it before with `existingLabel` check.

What do you think?

Reply:
makes perfect sense. thanks!